kpt-backstage-plugins/backstage.json

```diff
@@ -0,0 +1,3 @@
+{
+  "version": "1.1.1"
```

Comment:
Good question. This file is generated by `yarn backstage-cli versions:bump` to indicate the current version of Backstage app, so this will differ from the versions that the plugins have.

https://backstage.io/docs/getting-started/keeping-backstage-updated